include/at_cmd.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef AT_CMD_INTERFACE_H_
+#define AT_CMD_INTERFACE_H_
+
+/**
+ * @file at_cmd_interface.h
+ *
+ * @defgroup at_cmd_interface AT Command interface driver
+ *
+ * @{
+ *
+ * @brief Public APIs for the AT command interface driver.
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <zephyr/types.h>
+
+/**
+ * @brief AT command return codes
+ */
+enum at_cmd_rc {
+	AT_CMD_OK,
+	AT_CMD_ERROR,
+	AT_CMD_CMS,
```

Comment:
CMS and CME should be treated as an error. Could you rename the fields to `AT_CMD_ERROR_CMS` and `AT_CMD_ERROR_CME` to make it explicit?

Reply:
ok